### ISSUE TITLE
Listener event expired key and dispatch actions?

### ISSUE THREAD

**Issue by DeeSouza** [luin/ioredis#1105]
I want that when Redis deletes a record when the expiration time expires, an action in NodeJS is invoked.

It is possible?

**Comment by luin** [comment]
Yeah, ioredis supports keyspace notifications via `redis.psubscribe`: https://redis.io/topics/notifications#timing-of-expired-events.
